### DISCUSSION TITLE
need to "complete user profile"

### DISCUSSION THREAD

**Discussion by oscie57** [community/community#57489] [GitHub Education]
### Hi everyone,

I have been allowed to re-apply, yay! Though, now another issue lies:

> Your application cannot be reviewed until you fix the following:
> - You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

As any of you can see when clicking upon my profile, my full name is shown on both my display name and readme, and my name is mentioned in my short bio. I have no clue what I have to do now, anyone else had this issue and figured out what to do?

**Comment by samus-aran** [comment]
Hi @oscie57 👋 

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the same [comment in multiple places](https://github.com/orgs/community/discussions/?discussions_q=involves%3Aoscie57). As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
